ports/bde/portfile.cmake

```
@@ -1,133 +1,72 @@
 vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
-set(CONFIGURE_COMMON_ARGS ${CONFIGURE_COMMON_ARGS} --library-type=static)
 
-set(BDE_VERSION 3.2.0.0)
-set(BDE_TOOLS_VERSION 1.x)
+set(BDE_VERSION 3.117.0.0)
+set(BDE_TOOLS_VERSION 3.117.0.0)
```

```suggestion
set(BDE_TOOLS_VERSION "${VERSION}")
```

Unless this as a separate concept no longer need exist?

---

I'll leave this variable here for now and just set it using VERSION as you suggest. Perhaps in future these versions will diverge?